src/main/java/third/logic/Main.java

```diff
@@ -0,0 +1,27 @@
+package third.logic;
+
+import third.view.InputView;
+import third.view.ResultView;
+
+import java.util.ArrayList;
+
+
+public class Main {
+
+    public static void main(String[] args) {
+
+        int raceCount = InputView.inputRaceCount();
+        int carCount = InputView.inputCarCount();
+
+        for(int i =0; i < carCount; i++) {
+            Car car = new Car();
+        }
+
+        ArrayList<Car> carArrayList = Car.getCarArrayList();
+
+        for( int i = 0; i < raceCount; i++){
```

Comment:
```suggestion
        for (int i = 0; i < carCount; i++) {
            Car car = new Car();
        }

        ArrayList<Car> carArrayList = Car.getCarArrayList();

        for (int i = 0; i < raceCount; i++) {
```

포맷팅에 신경 써 주세요~ [단축키](https://www.jetbrains.com/help/idea/reformat-and-rearrange-code.html) 사용을 습관화하시면 좋습니다.